package.json

```diff
@@ -80,7 +80,7 @@
     "start-static": "npm run compile:prod && terra express-server --site build",
     "test": "npm run lint && npm run jest && npm run wdio",
     "test:docker": "npm run lint && npm run jest && npm run wdio:docker",
-    "wdio": "terra wdio --themes terra-default-theme clinical-lowlight-theme orion-fusion-theme",
+    "wdio": "terra wdio --themes terra-default-theme clinical-lowlight-theme orion-fusion-theme --spec packages/terra-date-time-picker/tests/wdio/date-time-picker-mobile-spec.js",
```

Comment:
Revert this change

Reply:
Thanks done https://github.com/cerner/terra-framework/pull/1969/commits/5723e357fafd47dd891a3ecc9b57179b20709a08